CONTRIBUTING.md

```diff
@@ -3,10 +3,10 @@
 [starterkit-lessons][repo] is an open source project,
 and we welcome contributions of all kinds:
 
-* New lessons;
-* Fixes to existing material;
-* Bug reports; and
-* Reviews of proposed changes.
+* New lessons
+* Fixes to existing material
+* Bug reports
+* Reviews of proposed changes
```

Comment:
You don't like this style of writing lists?

Reply:
No--the connection between the thoughts is implied by the bullets, making the extra punctuation and 'and' redundant.

Reply:
OK. This is a stylistic choice that I disagree with, but don't hold the opinion very strongly. Either way, we should try to be consistent with the formatting in other lists elsehwere.

Reply:
I believe this is the only list that had the `; and` construction, though I could be wrong. In any case, I agree we should be consistent.